regulations/templates/regulations/preamble-chrome.html

```diff
@@ -56,7 +56,7 @@ <h3 class="subpart-heading">{{cfr_part_toc.title}} CFR {{cfr_part_toc.part}}</h3
 {% block chrome-content %}
 <main role="main">
 
-  <div id="content-body" class="main-content">
+  <div id="content-body" class="main-content preamble-wrapper">
```

Comment:
This class was just removed in #217 -- would it be worth a short convo to confirm you and @jmcarp are on the same page?